### PR TITLE
Cleanups and hotfixes

### PR DESCRIPTION
**Pull request by vladimirsouza**

### CODE REVIEW

File: R/fancy_circos_plot.R
Comment (vladimirsouza) [inline R/fancy_circos_plot.R:1]: In `fancy_circos_plot`, `svs_df` data frame is created
```
> svs_df
  CHROM_A   START_A     END_A CHROM_B   START_B     END_B                  manta_name
1      14 106329465 106329465      18  60793497  60793497   MantaBND:194451:1:2:0:0:0
2      14 106379091 106379091      18  60793492  60793492   MantaBND:194451:0:1:0:0:0
3       8 128748200 128748200      14 106114286 106114286 MantaBND:135279:0:1:0:0:0:0
4       8 128748204 128748205      14 106114282 106114283 MantaBND:135279:0:1:0:0:0:0
```
Then the code splits `manta_name` column into several columns. However, rows may have different number of columns
```
>     #split manta_name variable
>     svs_df = data.frame(svs_df$CHROM_A, svs_df$START_A, svs_df$END_A, svs_df$CHROM_B, svs_df$START_B, svs_df$END_B, do.call(rbind, strsplit(svs_df$manta_name, split = ":", fixed = TRUE)))
Warning message:
In (function (..., deparse.level = 1)  :
  number of columns of result is not a multiple of vector length (arg 1)

> svs_df
  svs_df.CHROM_A svs_df.START_A svs_df.END_A svs_df.CHROM_B svs_df.START_B svs_df.END_B       X1     X2 X3 X4 X5 X6 X7       X8
1             14      106329465    106329465             18       60793497     60793497 MantaBND 194451  1  2  0  0  0 MantaBND
2             14      106379091    106379091             18       60793492     60793492 MantaBND 194451  0  1  0  0  0 MantaBND
3              8      128748200    128748200             14      106114286    106114286 MantaBND 135279  0  1  0  0  0        0
4              8      128748204    128748205             14      106114282    106114283 MantaBND 135279  0  1  0  0  0        0
```

Later the code uses `X1` column (renamed to `TYPE`) but never uses the columns `X2, X3, ..., X8`. So I just removed the columns `X2, X3, ..., X8`.

Parameters `from_flatfile` and `use_augmented_maf` were removed in [this](https://github.com/morinlab/GAMBLR.viz/commit/39942cf0e5a64da0d0af85fb0a6d9e3fd6e90c10#diff-034cbcc2cc20a6347d822a4307f746e1ded1c7783bb70e8ce9292e556f36ba2a) PR, but not from the documentation.

File: R/fancy_circos_plot.R [hunk 1]
```diff
@@ -163,7 +162,7 @@ fancy_circos_plot = function(this_sample_id,
     maf_tmp$Variant_Type = as.factor(maf_tmp$Variant_Type) #transform Variant_Type to factor
     maf_tmp[maf_tmp==0] <- 1 #transform all lenght coordinates == 0 to 1
 
-    if(!str_detect(maf_tmp$Chromosome, "chr")){ #add chr prefic, if missing...
+    if( any( !str_detect(maf_tmp$Chromosome, "chr") ) ){ #add chr prefic, if missing...
```
Comment (Kdreval) [inline R/fancy_circos_plot.R:165]: If you are editing this, can you please also fix the typo in `prefix`?

Comment (vladimirsouza) [reply R/fancy_circos_plot.R]: Done

**Review by vladimirsouza — COMMENTED** [review]
About change in file [R/fancy_propcov_plot.R](https://github.com/morinlab/GAMBLR.viz/pull/13/files#diff-766c94a570622bd789fab09e76456f811eb726ba07969bfcc11c4261852c0e7b)
See [this](https://github.com/morinlab/GAMBLR.viz/issues/14) Issue

**Review by Kdreval — APPROVED** [review]

**Pull request merged**